app/src/main/java/org/greenstand/android/TreeTracker/viewmodels/MapViewModel.kt

```diff
@@ -30,7 +28,7 @@ class MapViewModel constructor(
         if (validateCheckInStatusUseCase.execute(Unit)) {
             checkInStatusLiveData.postValue(true)
         } else {
-            expireCheckInStatusUseCase.execute(Unit)
+            userManager.clearUser()
```

Comment:
Can we use some other intuitive name.  Instead of `clearUser`, we could `logOff` or `expireCheckInStatus`?

Reply:
Good idea!